app/views/logs/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Displaying logs for...</h1>
+
+<ul>
+  <% @logs.each do |log| %>
+    <li><%= log["username"] %></li>
```

Comment:
Small but I always prefer hashes to blow up with .fetch() so pesky nils don't percolate through the system